Source/AutoExtern/Program.cs

```diff
@@ -398,10 +400,16 @@ public static class Program {
     return template;
   }
 
-  public static string GenerateDafnyCode(string projectPath, string filePath, string cSharpRootNS) {
-    var ast = AST.FromFile(projectPath, filePath, cSharpRootNS);
+  public static string GenerateDafnyCode(string projectPath, IList<string> sourceFiles, string cSharpRootNS) {
     var wr = new StringWriter();
-    ast.Pp(wr, "");
+    var asts = AST.FromFile(projectPath, sourceFiles, cSharpRootNS).ToList();
+    var last = asts.Last();
+    foreach (var ast in asts) {
+      ast.Pp(wr, "");
+      if (ast != last) {
+        wr.WriteLine();
+      }
+    }
```

Comment:
```suggestion
    foreach (var ast in asts) {
      ast.Pp(wr, "");
      wr.WriteLine();
    }
```

Reply:
Why this suggestion?

Reply:
It simplifies the code and it doesn't cost anything.  Not a big deal.

Reply:
Also, it's marginally prettier, in that it doesn't depend on the sequence being non-empty? 🤷

Reply:
That's nice. I didn't want to have a double line in the output that's why I made this change.

Reply:
OK, works for me :)